recipes/lsst-dd-rtd-theme/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "lsst-dd-rtd-theme" %}
+{% set version = "0.2.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://files.pythonhosted.org/packages/3f/de/c019bf966769b991d0595fd37434c2398608a4cc11ebfabe0d91ac8d7df5/lsst_dd_rtd_theme-0.2.2.tar.gz
```

Comment:
You need to use the url in the example recipe. Otherwise, the conda forge bots won’t be able to update this.